test/acceptance/behave/components/common_steps/entities/get_steps.py

```diff
@@ -65,6 +65,18 @@ def get_an_entity_by_id(context, entity_id):
     __logger__.info("...returned an entity by id")
 
 
+@step(u'get attributes in an entity by ID "([^"]*)"')
+def get_attributes_in_an_entity_by_id(context, entity_id):
```

Comment:
Remember for future PRs... it is better to split changes in PRs with .py changes and .feature changes.